bin/compile

```diff
@@ -1 +1,36 @@
 #!/bin/bash
+
+BUILD_DIR=$1
+CACHE_DIR=$2
+LP_DIR=`cd $(dirname $0); cd ..; pwd`
```

Comment:
This was a typo in the `apt` buildpack. It was meant to be `BP_DIR` as it points to the location of the buildpack itself.

It's documented in the buildpack docs that if you create a file `$BP_DIR/export` then that file gets treated as a script that will be run before later buildpacks. So we can dump items in that file that can set up later environment details.